From cd897d5113df06ddea089d8d5cb1e02e1a39aef2 Mon Sep 17 00:00:00 2001
From: S-trace <S-trace@list.ru>
Date: Sun, 8 May 2016 16:19:12 +0300
Subject: [PATCH 20/54] [BOOT] [INIT] Disable qseecomd service

---
 ramdisk/init.target.rc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ramdisk/init.target.rc b/ramdisk/init.target.rc
index 45c637d..565071b 100755
--- a/ramdisk/init.target.rc
+++ b/ramdisk/init.target.rc
@@ -140,6 +140,7 @@ service qseecomd /system/bin/qseecomd
     class late_start
     user root
     group root
+    disabled
 
 service audiod /system/bin/audiod
    class late_start
-- 
2.7.4

